components/cards/CardButton.jsx

```diff
@@ -42,14 +43,15 @@ const CardButton = ({
     if (onlyPerformAction === true) {
         return (
             <div className="card__button">
-                <button
+                {/* <button
                     className={`card__button--secondary ${variantMapper[variant]} ${customClassName}`}
                     aria-label={`${text}`}
                     disabled={disabled}
                     onKeyUp={(e) => handleKeyUp(e)}
                     onClick={action}>
                     {text || children}
-                </button>
+                </button> */}
+                <Button onKeyUp={(e) => handleKeyUp(e)} onClick={action} copy={text || children} buttonTitle={text || children} buttonSize="md" buttonType="primary" backgroundColor="light" /> 
```

Comment:
i would think you would need to pass in the buttonType here? are they all primary buttons?